examples/24-nodegroup-subnets.yaml

```diff
@@ -22,7 +22,7 @@ vpc:
         id: subnet-03a1260affe6f9944
     private:
       private-one:
-        id: subnet-05373e0e6d03065cc
+        cidr: "192.168.160.0/19"
```

Comment:
i actually like having all variations of a config thing in one file, but that could be because i am lazy.

maybe we could have comments? like `"you can alternatively do this"`?

Reply:
that sounds good